### DISCUSSION TITLE
Позёрство. квест дара про кольцо орков

### DISCUSSION THREAD

**Discussion by Kor-Angar** [dosinabox/g2nr_unofficial_update#358] [Ideas]
Часть Наёмников уходит охотится на драконов в конце 3 главы. Дар даёт квест "позерство" в конце 3 главы. а орк с кольцом появляется в конце 3 или с 4 главы?

**Comment by dosinabox** [comment]
С 4.

**Comment by Kor-Angar** [comment]
а квест даёт в конце 3 ?

**Comment by dosinabox** [comment]
да, после восстановления Глаза

**Comment by dosinabox** [comment]
В теории какой-нибудь трофей можно спрятать в 4 главе в Рудниковой долине. Но не кольцо. Сам Дар упоминает нашивку или знамя.
Можно запрятать или дать кому-нибудь орочий рог из Г1 - к нему остались все анимации и звуки, в Г2 в него можно дудеть)
Кажется, Ур-Шак становится местным главным в 4 главе?

**Comment by Kor-Angar** [comment]
> Можно запрятать или дать кому-нибудь орочий рог из Г1 - к нему остались все анимации и звуки, в Г2 в него можно дудеть)

можно взять Элитного Орка который стоит возле трупа Паладина, переодеть его в новые Доспех Орка военачальника, дать ему топор помощнее, усилить параметры и закинуть ему рог. 

Таким образом игрок обратит на него внимание, не пропустит рог и будет думать что с ним делать.

Если ты хочешь именно что то запрятать то есть место что то типа жертвенного алтаря, там лежит свиток превращение в Мракориса и ItMw_2H_Orcaxe_04.


> Кажется, Ур-Шак становится местным главным в 4 главе?

если его не убъёт герой во 2 главе.
